ci/warning/purge-ubuntu-runner.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+# !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+# WARNING: THIS IS NOT TO BE USED LOCALLY, BUT ONLY IN GITHUB ACTIONS TO CLEAR
+# DISK SPACE ON UBUNTU. IT IS NOT MARKED AS EXECUTABLE BY DEFAULT FOR SAFETY.
+# !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+sudo rm -rf /usr/share/dotnet /usr/local/lib/android /opt/ghc
```

Comment:
is this necessary with `dotnet-sdk-* mono-complete ghc-*` purges below?  any idea where the android stuff comes from?

Reply:
Good point, let's see what happens without this line. `/usr/local/lib/android` isn't installed with `android-sdk`, so honestly I have no idea. Let's roll the dice!